deploy/examples/v2/process_migration_externaldb.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: process-migration
+spec:
+  environment: rhpam-trial
+  objects:
+    processMigration:
+      image: "rhpam-process-migration-rhel8-mssql"
```

Comment:
what is this image? is it something RH will provide?

Reply:
This is an example image. It is supposed to be based on the process migration image and packaged with the jdbc driver for the external db connection. Please let me know if there is a  way to make it less ambiguous. Thanks.

Reply:
maybe add a commented out description above it?